src/formatHit.test.js

```diff
@@ -36,6 +36,23 @@ describe('formatHit', () => {
       }
     }),
     getTestCase({
+      name: 'locale_names[1] is matching',
+      hit: {
+        locale_names: ['Paris', 'Lutèce'],
+        city: ['Paris'],
+        administrative: ['Île-de-France'],
+        _highlightResult: {
+          locale_names: [{value: 'Paris', matchedWords: []}, {value: 'Lutèce', matchedWords: ['Lutèce']}]
+        }
+      },
+      expected: {
+        name: 'Paris',
+        administrative: 'Île-de-France',
+        city: undefined,
```

Comment:
> Test result says city is Paris here

Actually the issue was in the `highlight` attribute. I've updated the tests to make it more explicit.